app/service/url_shortener.py

```diff
@@ -0,0 +1,20 @@
+from app.models.database import create_link, get_link_by_short_url
+
+
+"""
+This file is a helper that recieves data from api_handlers 
+and converts it into the short_link format needed for the databse
+"""
+def create_shortened_url(original_url: str) -> str:
```

Comment:
There doesn't seem much reason to separate url_shortener.py from the database.py

I'll check the assignment specs to make sure I'm not giving conflicting info. 

But in this code, it makes more sense to make one file `url_storage.py` which combines these 2 files. The functions in these 2 files are the same level of depth in the code base. If highest level is the customer and lowest level is the database.

Reply:
I have a question about this, I'll make a video right now.